components/component.go

```diff
@@ -82,10 +86,11 @@ type ManifestsConfig struct {
 
 type ComponentInterface interface {
 	Init(ctx context.Context, platform cluster.Platform) error
-	ReconcileComponent(ctx context.Context, cli client.Client,
-		owner metav1.Object, DSCISpec *dsciv1.DSCInitializationSpec, platform cluster.Platform, currentComponentStatus bool) error
+	ReconcileComponent(ctx context.Context, cli client.Client, owner metav1.Object, DSCISpec *dsciv1.DSCInitializationSpec,
```

Comment:
not sure what the change is about. Reformatting? Why?

Reply:
I will not block the PR, but the questions are marked resolved without reverting the unrelated changes nor explanation.

Reply:
Sorry that wasn't intentional, it was auto-formatted for some reason. Now it is reverted. thanks